pvextractor/gui.py

```diff
@@ -243,16 +242,26 @@ def __init__(self, filename_or_cube, backend="Qt5Agg", clim=None):
             if self.array.ndim != 3:
                 raise ValueError("dataset does not have 3 dimensions (install the spectral_cube package to avoid this error)")
 
-        self.backend = backend
-
         import matplotlib as mpl
 
-        mpl.use(self.backend)
+        # We reset the rc parameters to the default values to make sure we use
+        # the default interactive backend and to make sure that the UI is
+        # consistent.
+        mpl.rcdefaults()
```

Comment:
will this change the global mpl defaults?  I don't think we want that as a default option since it could surprise users who have modified their defaults locally.

Reply:
Fixed - now I use a decorator for the methods to make sure we only use the default rc params for this GUI, not for user figures.